server/views/grid/aligning/README.md

```diff
@@ -1 +1,3 @@
 Applying a class of either `grid--start`, `grid--center` or `grid--end` to the containing `grid` div will determine how grid cells are aligned horizontally.
+
+Applying a class of `grid--spaced` to the containing `grid` element will space out its child `grid__cell`s (useful for when e.g. there is an empty column of space between to content columns, rather than having to include an empty element for spacing).
```

Comment:
Good question – I suppose it would in instances where it was used to space more than two columns (with only two, the first/last would be flush left/right, so all good – float left/right equivalent).